frontend/src/components/eth/TokenBalance.js

```diff
@@ -68,6 +70,12 @@ export const tokenSlice = createSlice({
       },
       allowanceDai: (state, action) => {
         state.allowanceDai = action.payload;
+      },
+      ratioToken: (state, action) => {
+        console.log(action)
+        console.log(state.decimalsDai)
+        state.ratioToken = decimalFormat(action.payload[0], state.decimalsEth) / decimalFormat(action.payload[1], state.decimalsDai)
+        console.log(state.ratioToken)
```

Comment:
this is actually useful log, lets keep it, maybe change it to:
```
console.log("ratio: ", state.ratioToken)
```
so we understand what is that number